.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Joshua must review all changes to deployment and build.sh
```

Comment:
don't think we need a comment here

Reply:
Maybe, maybe not, I think it's important in case anyone is curious why.